src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/index.ts

```diff
@@ -186,6 +212,17 @@ export const thorchainSaversStakingOpportunitiesUserDataResolver = async ({
 
       const userStakingId = serializeUserStakingId(accountId, stakingOpportunityId)
 
+      if (stakingOpportunityId === thorchainAssetId) {
+        stakingOpportunitiesUserDataByUserStakingId[userStakingId] = {
+          isLoaded: true,
+          userStakingId,
+          stakedAmountCryptoBaseUnit: '0',
+          rewardsCryptoBaseUnit: { amounts: ['0'], claimable: false },
```

Comment:
Add a TODO to implement me

Reply:
Added in [d387f46](https://github.com/shapeshift/web/pull/7430/commits/d387f46ef66e91fa6dc499cf57b03c1153695497)